src/roman_datamodels/datamodels.py

```diff
@@ -16,10 +16,10 @@
 from collections.abc import Iterable
 from pathlib import PurePath
 
-import asdf
-import numpy as np
-import packaging.version
-from astropy.time import Time
+import asdf  # type: ignore
+import numpy as np  # type: ignore
+import packaging.version  # type: ignore
+from astropy.time import Time  # type: ignore
```

Comment:
```suggestion
import asdf
import numpy as np
import packaging.version
from astropy.time import Time 
```
Left over from prototyping.